tfhe/src/shortint/oprf.rs

```diff
@@ -117,6 +121,80 @@ pub(crate) mod test {
         a * a
     }
 
+    #[test]
+    // This test is seeded which prevents flakiness
+    // The noise added by the KS and the MS before the PRF LUT evaluation can make this test fail
+    // if the seeded input is close to a boundary between 2 encoded values
+    // Using another KS key can, with a non-neglibgible probability,
+    // change the output of the PRF after decoding
+    fn oprf_compare_plain() {
```

Comment:
add _ci_run_filter at the end otherwise it will not run with our current test filters

Reply:
Done